tools/travis-build-and-push-docker.sh

```diff
@@ -1,9 +1,5 @@
 #!/bin/bash
 
-# travis secure envs are available only of our own PRs
```

Comment:
can we check that required variables are set and fail with concrete error at the beginning?
Instead of a random one in the middle of the test.

Reply:
Good idea, although it is not exactly in scope of this PR (new ticket perhaps?).

Reply:
Can be a separate PR, right.